### PR TITLE
feat: uniV3 basic lp info

### PR DESCRIPTION
**Pull request by 0xrusowsky**
this PR will tackle:
 - https://github.com/duneanalytics/spellbook/issues/5626

in order to ship a spell that tracks volumes served by liquidity managed by ALMs (active liquidity managers), it is essential to first have a snapshot of the historical liquidity on each pool.

As a first-time contributor, i take this PR as a test to start small and asses whether what i build breaks or not. So the spell `dex.lps` created in this PR doesn't necessarily need to make it to production, as that logic could be internalized inside the pipeline to get the data in `alm.trades`.

### CODE REVIEW

**Comment by 0xrusowsky** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/duneanalytics/spellbook/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I hereby sign the CLA
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request

I have read the CLA Document and I hereby sign the CLA

**Comment by 0xrusowsky** [comment]
recheck

**Comment by Hosuke** [comment]
> > **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/duneanalytics/spellbook/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> > I have read the CLA Document and I hereby sign the CLA
> > You can retrigger this bot by commenting **recheck** in this Pull Request
> 
> I have read the CLA Document and I hereby sign the CLA

You can sign the CLA without quoting/replying to the bot message.

File: macros/models/_sector/dex/uniswap_compatible_lp.sql [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                0
+            {% endif %} AS position_id
+        , b.tickLower AS tick_lower
+        , b.tickUpper AS tick_upper
+        , b.amount AS liquidity
+        , b.amount0
+        , b.amount1
+        , f.token0 AS token0_address
+        , f.token1 AS token1_address
+        , m.contract_address AS pool_address
```
Comment (Hosuke) [inline macros/models/_sector/dex/uniswap_compatible_lp.sql:92]:
```suggestion
        , b.contract_address AS pool_address
```

Comment (0xrusowsky) [reply macros/models/_sector/dex/uniswap_compatible_lp.sql]: thanks!
yesterday i had to do other stuff, but will continue working on this during this afternoon and next week

Comment (Hosuke) [reply macros/models/_sector/dex/uniswap_compatible_lp.sql]: No worries, if you have any problem please feel free to tag me.

Comment (0xrusowsky) [reply macros/models/_sector/dex/uniswap_compatible_lp.sql]: great, thanks for your support!

**Comment by Hosuke** [comment]
Also please sign the CLA, thank you @0xrusowsky

**Comment by 0xrusowsky** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by 0xrusowsky** [comment]
recheck

**Comment by 0xrusowsky** [comment]
@Hosuke shouldn't this work?
- https://github.com/duneanalytics/spellbook/pull/5635#issuecomment-2024680701

**Comment by Hosuke** [comment]
Not sure why CLA check is failed.

**Comment by jeff-dude** [comment]
recheck

**Comment by jeff-dude** [comment]
not 100% sure honestly. i'd say try one more time copy/paste the words exactly without quoting anything. then don't comment `recheck`, see if it works on it's own.

if not, maybe close the PR and start fresh with new PR, comment again with just the sentence

**Pull request closed without merging**